bkp/bkp_ekat_assert.hpp

```diff
@@ -0,0 +1,146 @@
+#ifndef EKAT_ASSERT_HPP
+#define EKAT_ASSERT_HPP
+
+#include <sstream>
+#include <exception>
+#include <assert.h>
+#include <stdexcept>  // For std::logic_error
+
+#include <cfenv>  // For FPE environment
+
+/*
+ * Asserts and error checking for Scream.
```

Comment:
These are comments, so not a big deal, but the word "scream" does appear here. :-)

Reply:
Ah, the person who invented uppercase...